<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vOutputerEx</class>
 <widget class="QWidget" name="vOutputerEx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>820</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblDate2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelect">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>551</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>551</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="VBar">
   <property name="geometry">
    <rect>
     <x>1155</x>
     <y>70</y>
     <width>41</width>
     <height>651</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
